adal/build.gradle

```diff
@@ -298,6 +297,7 @@ artifacts {
     archives sourcesJar
 }
 
+/*
```

Comment:
Do we plan to readd? Or should we remove outright?

Reply:
From offline mail, sounds like plan is remove this entirely and swap it out for SpotBugs -- in that case, I'm fine with deleting these blocks entirely, since there's no plan to reenable later

Reply:
I removed all references to findbugs.  I added the workitem to track introducing spotbugs to our projects.  https://identitydivision.visualstudio.com/Engineering/_boards/board/t/Auth%20Client%20-%20Android/Backlog%20items/?workitem=1151494